ChangeLog.md

```diff
@@ -11,3 +12,16 @@
 ### Bug fixes
 
 ### Miscellaneous
+
+## Version 0.1.1-dev 
```

Comment:
can the most recent notes be put at the top of the file so the user sees the most recent changes first? normally the flow is:

```text
## version 1.2

### Features

* this really awesome thing #123

### Bug Fixes

* this really nasty thing happened where blah blah, see #20 for more detail

## Version 1.1

...
```